components/footer.js

```diff
@@ -4,24 +4,10 @@ import { FaGithub } from 'react-icons/fa';
 import { FaTwitter } from 'react-icons/fa';
 import { FaInstagram } from 'react-icons/fa';
 import { Container } from './container';
-export const Footer = () => {
+export const Footer = (footer) => {
   return (
     <div className="footer">
-      <Container>
-        <h3>Göker ZAFER</h3>
-        <p>Hello,</p>
-        <p>I am a passionate frontend developer!</p>
-        <p>If you have an amazing project and want to work with me </p>
-        <p>please dm me in one of the social media platforms.</p>
-        <IconWithLink
-          link="https://www.linkedin.com/in/gokerzafer/"
-          icon={FaLinkedinIn}
-        />
-        <IconWithLink link="https://www.instagram.com/gokerlek" icon={FaInstagram} />
-        <IconWithLink link="https://github.com/gokerlek" icon={FaGithub} />
-        <IconWithLink link="https://twitter.com/gokerzfr" icon={FaTwitter} />
-        <h5>© 2021 - 2021 www.gokerzafer.com - All Rights Reserved.</h5>
-      </Container>
+      <Container footer={footer}></Container>
```

Comment:
you don't need to send the footer data to container here

Reply:
Instead you need to use footer data to generate the previous embedded elements. For instance:
```javascript
<h5>{footer.copyright}</h5>
```